test/app-tap/datetime.test.lua

```diff
@@ -1867,6 +1868,51 @@ test:test("totable{}", function(test)
     end
 end)
 
+test:test('totable{} with timezone', function(test)
```

Comment:
Nit: I don't understand the meaning of `{}` here.
Do you mean just totable instead?

Reply:
Sure, it should be `()` instead `{}`.
This is due to copypaste testcase above:

>  test:test("totable{}", function(test)

Fixed:

```diff
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -1868,7 +1868,7 @@ test:test("totable{}", function(test)
     end
 end)
 
-test:test('totable{} with timezone', function(test)
+test:test('totable() with timezone', function(test)
     local DEFAULT_TZOFFSET = 0
     local DEFAULT_TZ = ''
 
```